back/src/forms/forms.graphql

```diff
@@ -65,13 +120,21 @@ type Query {
     roles: [FormRole!]
 
     """
-    (Optionnel) Permet de filtrer sur les bordereaux en attente d'une action de notre part
+    (Optionnel) Permet de filtrer sur les bordereaux en attente d'une action de votre part
     Si `true`, seul les bordereaux attendant une action sont renvoyés
     Si `false`, seul les bordereaux n'attendant aucune action son renvoyés
     Si vide, tous les bordereaux sont renvoyés
     Défaut à vide.
     """
     hasNextStep: Boolean
+
+    """
+    (Optionnel) Siret d'une autre entreprise présente sur le bordereau
+    Vous n'avez pas besoin d'être membre de cette entreprise.
+    Seuls les bordereaux ou cette entreprise apparait (dans n'importe quel cadre) seront retournés.
+    Défaut à vide.
+    """
+    siretPresentOnForm: String
   ): [Form!]!
```

Comment:
Du coup on n'est pas complètement raccord avec la façon de faire sur `formsLifeCycle` où l'on renvoie un objet contenant les champs `hasNextPage`, `hasPreviousPage`, `startCursor`, `endCursor`, `count`. Il faudra faire l'effort d'homogénéiser tout ça lors de la conception des DASRI. J'ai l'impression que la façon de faire par défaut en GraphQL est d'utiliser en paramètres `first`, `after`, `last`, `before` et de retourner un objet de type *Connection qui contient une liste de `edges` et un objet `pageInfo`. 

Voir par exemple:
 - https://graphql.org/learn/pagination/#pagination-and-edges
- https://developer.github.com/v4/query/#connections

Mais du coup impossible à implémenter sans faire de breaking change sur cette query donc on n'a pas trop le choix

Reply:
Yes c'est le problème restant, et je ne sais pas trop comment le résoudre sans faire de GROS breaking change. FormsLifecycle neste les résultats dans une sous propriété.
C'est compliqué à mon sens de toucher à ce point la au retour de `forms`...

Reply:
Je suis d'accord, la pagination c'est typiquement le genre de truc qu'on va avoir un peu partout. Ça serait top d'avoir une approche cohérente. On est un peu limité sur ce qu'on peut faire aujourd'hui mais on pourra faire ça bien bientôt.

En attendant, on peut peut-être tendre à se rapprocher des recommandations pour que lors de la mise à niveau vers la nouvelle version les concepts soient familiers pour les utilisateurs de l'API. Je pense aux noms des arguments (par exemple `first` plutôt que `formsPerPage`, pour éviter d'avoir `dasriPerPage` ou `companiesPerPage` dans le futur).

Reply:
Le mieux est de coller aux recommandations amha, on pourra créer une nouvelle query qui remplacera formsLifecylce avant de la déprécier.

Reply:
Ok pour le naming, j'ai repris `first` et j'ai ajouté `last` du coup.
Pour le pageInfo on reste donc un peu en rade pour cet API mais on peut normaliser tout ça pour les prochaines